plugin/trino-ranger/src/main/java/io/trino/plugin/ranger/RangerConfig.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ranger;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+
+public class RangerConfig
+{
+    private String keytab;
+    private String principal;
+    private boolean useUgi;
+    private String hadoopConfigPath;
+    private String serviceName;
+    private String securityConfigPath;
+    private String auditConfigPath;
+    private String policyMgrSslConfigPath;
+
+    public String getKeytab()
+    {
+        return keytab;
+    }
+
+    @Config("ranger.keytab")
+    @ConfigDescription("Keytab for authentication against Ranger")
+    @SuppressWarnings("unused")
+    public RangerConfig setKeytab(String keytab)
+    {
+        this.keytab = keytab;
+        return this;
+    }
+
+    public String getPrincipal()
+    {
+        return principal;
+    }
+
+    @Config("ranger.principal")
+    @ConfigDescription("Principal for authentication against Ranger with keytab")
+    @SuppressWarnings("unused")
+    public RangerConfig setPrincipal(String principal)
+    {
+        this.principal = principal;
+        return this;
+    }
+
+    public boolean isUseUgi()
+    {
+        return useUgi;
+    }
+
+    @Config("ranger.use_ugi")
```

Comment:
How does it work? What do you mean by groups here?

Reply:
`UserGroupInformation` class in Hadoop library can be used to obtain groups for a user. Kerberized environments typically use this approach to find groups for a user - for consistency across multiple services. Configuration `ranger.use_ugi` enables a deployment to use this approach.

Reply:
As I understand it is no longer needed when you removed the Kerboros, right?